src/bin/copy.c

```diff
@@ -85,37 +86,32 @@ static char *getuser(void)
 
 /*
  * If UNIX user is in UNIX group of directory containing file,
- * return gid, otherwise -1
+ * return 1, otherwise 0.
```

Comment:
the return of the function is still a gid_t should be int.

Reply:
It "never" returned a gid_t, but yes your right. Fixed :-)